### ISSUE TITLE
DeprecationWarning about pkg_resources.declare_namespace

### ISSUE THREAD

**Issue by kba** [qurator-spk/dinglehopper#76]
https://github.com/qurator-spk/dinglehopper/blob/bd324331e637fbc4eeff3a9d95c2b94d58b15a28/qurator/__init__.py#L1

I'm getting

```
DeprecationWarning: Deprecated call to `pkg_resources.declare_namespace('qurator')`.
  Implementing implicit namespace packages (as specified in PEP 420) is preferred to `pkg_resources.declare_namespace`. See https://setuptools.pypa.io/en/latest/references/keywords.html#keyword-namespace-packages
```

for these, consider switching to using [implicit PEP420 namespaces](https://peps.python.org/pep-0420/#specification)?

**Comment by mikegerber** [comment]
(Edit to fix the link to PEP420)

**Comment by mikegerber** [comment]
Messing with this stuff feels like computer-aided Jenga, I hope it works 🙏

**Comment by mikegerber** [comment]
Yeah, it's broken... Should have tested it properly, as installation is broken now.

**Comment by mikegerber** [comment]
I guess it's OK now.

![no idea what i am doing](https://user-images.githubusercontent.com/34309482/224995041-0f7fffec-41ff-4b66-a1d5-a4632c7f31b3.gif)

**Comment by mikegerber** [comment]
https://github.com/OCR-D/ocrd_all/pull/356#issuecomment-1474492390

The GIF couldn't be truer. It seems this is broken because i.e. eynollah (and probably other packages) uses the old style namespace.

**Comment by mikegerber** [comment]
Thinking about just ditching the qurator namespace and using "dinglehopper".

**Comment by mikegerber** [comment]
Reinstalling it makes it broken again...

**Comment by mikegerber** [comment]
#77 has more info of why I didn't notice further breakage. 

I'm done for the week and going to fix this on Monday.

**Comment by mikegerber** [comment]
Should be fixed with f668963a2e1fa2f83e44c9dcca666cb98c9a80af
